.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
```

Comment:
I feel the language for the feature request is a bit odd here. If someone is requesting a feature, it might not be related to a problem. I would expect that the "feature request" section would come before asking if it is related to a current problem or issue with the project.

Reply:
I've updated the text to be more clear. This is to determine if the feature request is related to an existing complaint about the project or if it's a new idea altogether. For example, a feature request that might be solving a problem would be something like: "The search terms required to access some fields are extremely long and it's frustrating to type them all out." and the feature request would be "Simplify search operators" and alternatives might be "Create an interface that allows me to select from a list of all operators and then define what I'm searching for that way"